codebase/config/sync/migrate_plus.migration.idc_ingest_new_items.yml

```diff
@@ -138,154 +188,191 @@ process:
     bundle: copyright_and_use
     bundle_key: vid
     value_key: name
-  # FIXME: test data references a Person and Corporate Body, only one can be resolved
```

Comment:
Just an FYI, it looks like some auto-formatting happened, and comments with TODOs were removed...

Reply:
This came directly from Drupal -- a make config-import and config-export will remove any comments and clean up the file to it's expected format. Not sure how to maintain these through an import/export of config.

Reply:
Resolving!